Dockerfile

```diff
@@ -33,3 +33,6 @@ RUN vca-install-package \
   bzip2 \
   gzip \
   xz-utils
+
+ADD entrypoint.sh /opt/entrypoint.sh
+ENTRYPOINT ["/bin/sh", "/opt/entrypoint.sh"]
```

Comment:
@mattyclarkson you're right, @cdbishop is going to revert this and then we'll essentially split what was the `entrypoint.sh` into two ci-scripts: one for starting sshd and another for installing the public keys (so the tests can run vca-tool-chain-upload to itself).